### PR TITLE
Remove dependency on `react-primitives`

### PR DESCRIPTION
**Pull request by hedvigoscar**
and try to hack it with index.{platform} files instead!

I havent actually tried it out with the new metro though but I guess it should work?

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { colors } from './colors'
-import { fonts } from './fonts'
+import { fonts } from './fonts/web'
```
Comment (Iamsamwhoami) [inline index.ts]: i dont think this will work, metro bundler would be trying to resolve either `fonts/web.ts` or `fonts/web.platform.ts`.

Comment (hedvigoscar) [reply index.ts]: Will the metro bundler not use `index.{platform}.js` from dist? 🤔

Comment (hedvigoscar) [reply index.ts]: I will test it out myself :+1:

Comment (Iamsamwhoami) [reply index.ts]: ah, yes, I got a bit confused by the structure, your approach should work.

I would prefer this actually:

- index.ts
- colors/index.ts
- fonts/index.android.ts goes to fonts/index.native.ts
- fonts/index.ios.ts goes to fonts/index.native.ts
- fonts/index.native.ts
- fonts/index.ts

Comment (hedvigoscar) [reply index.ts]: Agreed, I refactored it to be like that!

**Comment by hedvigoscar** [comment]
@Iamsamwhoami done!

**Comment by Iamsamwhoami** [comment]
:shipit:

**Comment by hedvigoscar** [comment]
Tested with `react-native@0.57`, looks like it works!

**Pull request merged**